types/src/v0/header.rs

```diff
@@ -76,8 +76,8 @@ impl Header {
                 ns_table,
                 block_merkle_tree_root,
                 fee_merkle_tree_root,
-                fee_info,
-                builder_signature,
+                fee_info: vec![fee_info],
+                builder_signature: vec![builder_signature],
```

Comment:
Since we now have multiple builders, would it make sense to make the builder signature part of the fee info type?

Reply:
There is discussion around this on zulip: https://espresso.zulipchat.com/#narrow/stream/440749-Marketplace/topic/Multiple.20fees.20per.20block/near/448326448